nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -174,6 +175,19 @@ public class ValidateCsv extends AbstractProcessor {
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor GET_ALL_VIOLATIONS = new PropertyDescriptor.Builder()
+            .name("validate-csv-violations")
+            .displayName("Get all violations")
+            .description("If true, the validation.error.message attribute would contain the list of all the violations"
+                    + " for the first invalid line. Note that setting this property to true would slightly decrease"
+                    + " the performances as all columns would be validated. If false, a line is invalid as soon as a"
+                    + " column is found violating the specified constraint and only this violation for the first invalid"
+                    + " line will be indicated in the validation.error.message attribute.")
+            .required(true)
+            .allowableValues("true", "false")
+            .defaultValue("false")
+            .build();
```

Comment:
Verbs such as `get` are usually for methods. In my opinion I think `ALL_VIOLATIONS` conveys the same point.
```suggestion
    public static final PropertyDescriptor ALL_VIOLATIONS = new PropertyDescriptor.Builder()
            .name("validate-csv-violations")
            .displayName("All violations")
            .description("If true, the validation.error.message attribute would contain the list of all the violations"
                    + " for the first invalid line. Note that setting this property to true would slightly decrease"
                    + " the performances as all columns would be validated. If false, a line is invalid as soon as a"
                    + " column is found violating the specified constraint and only this violation for the first invalid"
                    + " line will be indicated in the validation.error.message attribute.")
            .required(true)
            .allowableValues("true", "false")
            .defaultValue("false")
            .build();
```

Reply:
I've not looked at all of the codebase but usually a True/False property's description is starting with a verb. I'm fine using another work like "check" but I do think a verb should be used.

Reply:
I agree that `Get` is not optimal, but `All Violations` seems unclear. What about `Include All Violations`? Perhaps for better clarity, even though there are only two options, what about `Error Reporting Strategy` that is either `ALL` or `FIRST`?

Reply:
`Include All Violations` sounds sufficient.

Reply:
@pvillard31 I am fine with include also.